lib/src/executables/dependency_validator_ignore.dart

```diff
@@ -0,0 +1,164 @@
+    ..addFlag('verbose',
+        abbr: 'v',
+        negatable: false,
+        help: 'Outputs all logging to stdout/stderr.')
+    ..addFlag(
+      'yes-to-all',
+      negatable: false,
+      help: 'Forces all patches accepted without prompting the user. '
+          'Useful for scripts.',
+    )
+    ..addSeparator('Dependency Validator Updater Options')
+    ..addOption(_dependency,
+        abbr: 'd',
+        mandatory: true,
+        help: 'The dependency that should be ignored.');
+
+  final parsedArgs = parser.parse(args);
+
+  if (parsedArgs['help'] == true) {
+    stderr.writeln(parser.usage);
+    return;
+  }
+
+  final logger = Logger('over_react_codemod.dependency_validator_ignore');
+
+  logger.onRecord.listen((rec) {
+    // ignore severe messages here because stderr will print it
+    if (rec.message.isNotEmpty && rec.level != Level.SEVERE) {
+      print('[${rec.level}] ${rec.message}');
+    }
+    if (rec.error != null) {
+      print(rec.error);
+    }
+    if (rec.stackTrace != null) {
+      print(rec.stackTrace);
+    }
+  });
+
+  final dependencyToUpdate = parsedArgs[_dependency] as String;
+  final pubspecs = pubspecYamlPaths();
+
+  var exitCode = 0;
+
+  var detectedV1 = false;
+
+  for (final pubspec in pubspecs) {
+    final pubspecFile = File(pubspec);
+    final VersionRange? dependencyValidatorVersion =
+        getNonHostedDependencyVersion(
+            pubspecFile.readAsStringSync(), 'dependency_validator');
+    final majorVersion = dependencyValidatorVersion?.min?.major;
+
+    // This means that either there is no dependency on `dependency_validator`
+    // or the version couldn't be parsed.
+    if (majorVersion == null) continue;
+
+    // If the script finds that one pubspec uses V1, we want to skip re-running the script.
+    // It appears that most libraries with multiple packages use the same version of
+    // dependency validator, and if that version happens to be V1, then looping through
+    // the packages will be redundant (and take a long time) because V1 already
+    // searches all nested files.
+    if (detectedV1) continue;
+
+    if (exitCode == 1) continue;
+
+    logger.info(
+        'Detected dependency_validator version $dependencyValidatorVersion');
+
+    // It is necessary to check the major version and handle each one
+    // differently because each major has a different method to identify
+    // dependencies to ignore.
+    switch (majorVersion) {
+      case 1:
+        detectedV1 = true;
+        exitCode = await runInteractiveCodemod([
+          ...filePathsFromGlob(Glob('**.yaml', recursive: true)),
+          ...filePathsFromGlob(Glob('**.dart', recursive: true)),
+          ...filePathsFromGlob(Glob('**.sh', recursive: true)),
+          ...filePathsFromGlob(Glob('**.txt', recursive: true)),
+          ...filePathsFromGlob(Glob('**Dockerfile', recursive: true)),
+        ], V1DependencyValidatorUpdater(dependencyToUpdate));
```

Comment:
The default codemod args like `--yes-to-all` aren't getting passed through so something like this should be added to all the `runInteractiveCodemod` calls.
```suggestion
        ], V1DependencyValidatorUpdater(dependencyToUpdate),
        // Only pass valid low level codemod flags
          args: args.where((arg) => !arg.contains(_dependency) && !arg.contains('d')),
        );
```

Reply:
Ugh, I completely forgot about that 🤦 Thank you for catching that!!